marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -129,55 +143,48 @@ class
   unsubscribeFromPlutusApp :: PlutusAppId -> m Unit
   unsubscribeFromWallet :: WalletId -> m Unit
 
+fetchWalletDetails
+  :: forall m r
+   . ManageMarlowe m
+  => WalletNickname
+  -> WalletInfo
+  -> ExceptT (Variant (ClientServerErrorRow r)) m WalletDetails
+fetchWalletDetails walletNickname walletInfo = withExceptT clientServerError do
+  let
+    walletId = view _walletId walletInfo
+  -- Get all plutus contracts associated with the restored wallet.
+  plutusContracts <- ExceptT $ PAB.getWalletContractInstances walletId
+  -- If we already have the plutus contract for the wallet companion and marlowe app
+  -- let's use those, if note activate new instances of them
+  { companionAppId, marloweAppId } <- ExceptT $
+    activateOrRestorePlutusCompanionContracts walletId plutusContracts
+  -- TODO (as part of SCP-3360):
+  --   create a list of "loading contracts" with the plutusContracts of type MarloweFollower
```

Comment:
This part doesn't make sense for a newly created wallet... it doesn't hurt either, it will always return an empty list of plutusContract and we will create the MarloweApp and WalletCompanion.